### PR TITLE
fix: fix translation key typos

### PR DESCRIPTION
**Pull request by napilla**
Asana tasks:
[Your Verification Code page bug](https://app.asana.com/0/1204871882394498/1208078375881247/f)
[Enter Verification Code page bug](https://app.asana.com/0/1204871882394498/1208078352660411/f)

The strings needed for these pages were translated, but the language translation keys have some typos in them, so there were never persisted to the pages. 

This PR fixes that. I was able to test the Email verification page, but not the page where the email sends the code and we return to enter it. 
image

### CODE REVIEW

**Review by krisrjohnson21 — DISMISSED** [review]
🧼

**Review by lukecamelo — DISMISSED** [review]
good stuff

File: files/templates/mbta/email/messages/messages_pt.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 secondFactor.emailSubject=Código de verificação MBTA
 secondFactor.emailbody=Seu código de verificação é: {0}
```
Comment (paulswartz) [inline files/templates/mbta/email/messages/messages_pt.properties]: question: should this be `emailBody` (capital-B)?

File: files/templates/mbta/login/messages/messages_pt.properties [hunk 2]
```diff
@@ -42,12 +42,12 @@ invalidPasswordRegexPatternMessage=Senha: não corresponde ao (s) padrão (s) re
...
 
-secondfactor.userNotFound=O usuário que enviou o e-mail de segundo fator não foi encontrado
+secondFactor.userNotFound=O usuário que enviou o e-mail de segundo fator não foi encontrado
 secondFactor.sendEmailError=Desculpe, houve um problema ao enviar o e-mail do segundo fator
 secondFactor.validateCodeTitle=Insira o código de verificação
-secondfactor.code=Insira o código de verificação única do seu e-mail.
-secondfactor.badeMailCode=Código único incorreto
-secondfactor.userNoEmail=O usuário não tem e-mail
+secondFactor.code=Insira o código de verificação única do seu e-mail.
+secondFactor.badeMailCode=Código único incorreto
```
Comment (paulswartz) [inline files/templates/mbta/login/messages/messages_pt.properties]: question: this disagrees with `messages_es`, which has `secondFactor.badEmailCode`. which one is right?

File: files/templates/mbta/login/messages/messages_zh_CN.properties [hunk 3]
```diff
@@ -42,14 +42,14 @@ invalidPasswordRegexPatternMessage=密码：无法匹配正则表达式模式。
...
-secondfactor.usernoEmail=用户没有电子邮件
-secondfactor.dosubmit=登录
-secondfactor.codeExpired=代码已过期
+secondFactor.usernotFound=未找到要发送第二因子电子邮件的用户
+secondFactor.sendEmailerror=抱歉，发送第二因素电子邮件时出现问题
+secondFactor.validateCodeTitle=输入验证码
+secondFactor.code=输入电子邮件中的一次性验证码。
+secondFactor.bademailcode=一次性代码不正确
+secondFactor.usernoEmail=用户没有电子邮件
+secondFactor.dosubmit=登录
```
Comment (paulswartz) [inline files/templates/mbta/login/messages/messages_zh_CN.properties]: question: a bunch of these look wrong (all lower-case). did we double check that they work?

File: files/templates/mbta/email/messages/messages_pt.properties (hunk 1 above)
Comment (napilla) [reply files/templates/mbta/email/messages/messages_pt.properties]: yes, fixed in https://github.com/mbta/keycloak-deploy/pull/37/commits/9f6315ec968af070bffadddb5650032f7af38f55

File: files/templates/mbta/login/messages/messages_pt.properties (hunk 2 above)
Comment (napilla) [reply files/templates/mbta/login/messages/messages_pt.properties]: It is `badEmailCode`, corrected in https://github.com/mbta/keycloak-deploy/commit/9f6315ec968af070bffadddb5650032f7af38f55

File: files/templates/mbta/login/messages/messages_zh_CN.properties (hunk 3 above)
Comment (napilla) [reply files/templates/mbta/login/messages/messages_zh_CN.properties]: fixed in https://github.com/mbta/keycloak-deploy/commit/9f6315ec968af070bffadddb5650032f7af38f55, thanks

**Review by paulswartz — APPROVED** [review]

**Pull request merged**